test/ipld-eth-star.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+const rlp = require('rlp')
+const BlockService = require('ipfs-block-service')
+const cidForHash = require('ipld-eth-block/src/common').cidForHash
```

Comment:
This is an anti-pattern, let's make avoid using paths and dots in conjunction. (preferably just dots)

Also, seems that cidForHash is used to calculate all the CIDs, can we have this as a module on npm `eth-hash-to-cid`, ideally even exports a CLI so that it is super easy to use.